src/server/pkg/deploy/cmds/cmds.go

```diff
@@ -138,6 +138,9 @@ func DeployCmd(noMetrics *bool) *cobra.Command {
 				return fmt.Errorf("volume size needs to be an integer; instead got %v", args[6])
 			}
 			manifest := &bytes.Buffer{}
+			if strings.Contains(args[2], "/") {
+				return fmt.Errorf("invalid Amazon secret access key; key cannot contain a slash '/'")
+			}
```

Comment:
Turns out no -- this was a different bug. One that happened manually (by human) and via a bug in my deploy script. So I'm closing this PR.